src/util.c

```diff
@@ -124,16 +124,12 @@ u8 CreateInvisibleSpriteWithCallback(void (*callback)(struct Sprite *))
     return sprite;
 }
 
-void StoreWordInTwoHalfwords(u16 *h, u32 w)
-{
-    h[0] = (u16)(w);
-    h[1] = (u16)(w >> 16);
+void StoreWordInTwoHalfwords(u16 *h, u32 w) {
+  h[0] = (u16)(w);
+  h[1] = (u16)(w >> 16);
 }
 
-void LoadWordFromTwoHalfwords(u16 *h, u32 *w)
-{
-    *w = h[0] | (s16)h[1] << 16;
-}
+void LoadWordFromTwoHalfwords(u16 *h, u32 *w) { *w = h[0] | (s16)h[1] << 16; }
```

Comment:
I haven't checked everything, but did you intend to reformat code like in this example?

Reply:
This reformatting (and others) are on accident, VSCode just really wants to format stuff. I will revert the formatting changes.